guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -254,53 +267,47 @@ inputElement.focus();
 ```
 
 This code needs to run after the element is rendered.
+The simplest way to accomplish this is by using a modifier.
 
-The simplest way to accomplish this is by using the `did-insert` modifier from [@ember/render-modifiers][render-modifiers].
-
-[render-modifiers]: https://github.com/emberjs/ember-render-modifiers
-
-```handlebars {app/components/edit-form.hbs}
-<form>
-  <input {{did-insert this.focus}}>
-</form>
-```
+Ember apps ship with [ember-modifier](https://github.com/ember-modifier/ember-modifier), an official part of the framework that provides a friendly API for writing your own element modifiers.
+In the following examples, the modifier API is imported from the `ember-modifier` package.
 
-```js {app/components/edit-form.js}
+```gjs {app/components/edit-form.gjs}
 import Component from '@glimmer/component';
-import { action } from '@ember/object';
+import { modifier } from 'ember-modifier';
 
 export default class EditFormComponent extends Component {
-  @action
-  focus(element) {
+
+  focusElement = modifier((element) => {
```

Comment:
This example will errror.

This modifier should be defined outside the component as a `const` declaration, or the template should say `{{this.focusElement}}`